common/src/models/user.rs

```diff
@@ -2,9 +2,12 @@ use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 use strum::{Display, EnumString};
 
+pub type UserId = String;
```

Comment:
This type alias is just for readability, right? Should we add a doc comment explaining the format of the ID (`user_<ULID>`).